src/Pages/LikedVideosPage/LikedVideosPage.jsx

```diff
@@ -0,0 +1,42 @@
+import React, {useState, useEffect} from 'react';
+import { getLocalStorage } from '../../Helpers/Helpers/Utils';
+import axios from 'axios';
+import { VideoCard } from '../../Shared';
+
+const LikedVideosPage = () =>{
+
+    const auth = getLocalStorage("authData");
```

Comment:
fix: At many places you are using encodedToken for getting this variable, and at many places you are using auth for getting the same variable from localstorage.

I would recommend stick to the same variable for getting the item from localstorage

Reply:
ok!